lib/windows/launcher/components/AppIcon.jsx

```diff
@@ -76,11 +76,13 @@ function renderNotice(app) {
     return null;
 }
 
-const AppIcon = ({ app }) => (
-    <div className="core-app-icon">
+const AppIcon = ({ app, onDragEnd }) => (
+    <div className="core-app-icon" draggable onDragEnd={event => onDragEnd(event)}>
```

Comment:
`onDragEnd={event => onDragEnd(event)}` would be equivalent to `onDragEnd={onDragEnd}`, in fact latter would be one less call.